beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -344,42 +344,39 @@ func sortedBlockWithVerifiedBlobSlice(blocks []interfaces.ReadOnlySignedBeaconBl
 }
 
 func blobRequest(bwb []blocks2.BlockWithVerifiedBlobs, blobWindowStart primitives.Slot) *p2ppb.BlobSidecarsByRangeRequest {
-	if len(bwb) == 0 {
```

Comment:
even though the call to `lowestSlotNeedsBlob` will return early in this case, I feel it's still safer to return early here in case that the helper changes and then we are faced with a call to `bwb[-1]` below

Reply:
fair enough